WIP started handbook v2 content (#6546)
PostHog/posthog.com: contents/handbook/making-users-happy.md

---
title: How we make users happy
sidebar: Handbook
showTitle: true
---

## How we build a product

> First, someone internally will suggest an idea. Usually this will come from James and Tim, but it has just as frequently come from anyone else on the team.

Reviewer: ```suggestion
First, someone internally will suggest an idea. Sometimes this will come from James and Tim, but it has just as frequently come from anyone else on the team.
```

If it requires a new team to build it – which it usually will – we'll start by hiring an ex-founder who is technical. We'll onboard them into the existing team that has the most overlap. This helps get them used to working with our codebase, and with the culture we look for from each team.

That person builds the MVP, and the only goal is to figure out if anyone will use it. With some products, the MVP may have more scope if we feel especially confident. Once the new product is in a non-embarassing state (that won't harm our brand), we add pricing to it and position it on our website. This drives much more demand. At this stage, the goal is to get the product to product-market fit in PostHog's platform, which means working with customers until we have 5 delighted, paying customers. 

> Once all this is done – which we'd expect to take a few months – we can start to innovate. This usually means some kind of platform play - for example, extending the product to enhance everything else we're working on, or shipping a new product that'd work well with it.

Reviewer: ```suggestion
Once all this is done – which we'd expect to take a few months – we can start to innovate. This usually means some kind of platform play - for example, extending the product to enhance everything else we're working on, or shipping another new product that'd work well with it.
```

## Engineers talk to users and provide support

> If you are working on a product at PostHog that hasn't hit product-market fit, then you should be _as close as possible_ to  your users, feeling whatever they feel so you have as much signal as possible to make the product great.
> 
> _Only once_ this has happened do we have centralized support. However, even once you're at this stage, this support is always from a developer so we don't needlessly triage problems from one place to another before they wind up in an engineer's hands anyway. We'd rather fewer people with deeper technical knowledge who can fix bugs and update docs as they go, since in the long run that will lead to the best product _and_ the lowest support cost, especially as our users are technical and problem solving is part of their nature.
> 
> No matter what stage a product, our engineers work hand in hand with our customers (also engineers) whenever they are building something.

Reviewer: This feels overly complex.  Maybe just
```suggestion
You should be _as close as possible_ to  your users, feeling whatever they feel so you have as much signal as possible to make the product great.

For established products with a lot of usage questions (how do I create an insight that does X for example), Customer Success helps with support.
```
